src/fixes.rs

```diff
@@ -130,13 +132,13 @@ mod tests {
     fn run_with_unfixable_warning_test() {
         let mut lines = vec![
             line_entry(1, 3, "A=B"),
-            line_entry(2, 3, "C"),
+            line_entry(2, 3, "UNFIXABLE-"),
             blank_line_entry(3, 3),
         ];
         let mut warnings = vec![Warning::new(
             lines[1].clone(),
-            "KeyWithoutValue",
-            String::from("The C key should be with a value or have an equal sign"),
+            "Unfixable",
```

Comment:
I think that in the future we should remove this test (when all checks will have `Fixer`).
At the moment use unreachable check name is ok :)

Reply:
Yes, you're right 👍